unittests/testanchorevents.py

```diff
@@ -211,10 +211,10 @@
         "conformance_report": "blobs/e2a1d16c-03cd-45a1-8cd0-547189df1273",
     },
     "principal_accepted": {
-        "display_name": "stunt-idp@jitsuin.com",
-        "email": "stunt-idp@jitsuin.com",
-        "issuer": "stunt-idp@jitsuin.com",
-        "subject": "stunt-idp@jitsuin.com",
+        "display_name": "stunt-idp@rkvst.com",
```

Comment:
what is this used for?  I would rather we did not have "stunt-idp" in here at all